jobs/src/main/scala/dcos/metronome/utils/glue/MarathonImplicits.scala

```diff
@@ -105,16 +105,20 @@ object MarathonImplicits {
     def toRunSpec: RunSpec = {
       val jobSpec = run.jobSpec
 
+      // apply overrides to config from the JobRunSpec
+      val env = jobSpec.run.env.to[Seq] ++ run.overrides.env
```

Comment:
should we somewhere validate that overrides does not introduce new variables? That sounds like a bug to me in any case...